readthedocs/proxito/views/serve.py

```diff
@@ -178,22 +180,18 @@ def get(self, request, proxito_path, template_name='404.html'):
             filename=kwargs.get('filename', ''),
         )
 
-        # Always add a `/` to the filename to match our old logic:
+        # ``filepath`` is the path without ``/<lang>/<version>`` and without
+        # query, starting with a ``/``. This matches our old logic:
         # https://github.com/readthedocs/readthedocs.org/blob/4b09c7a0ab45cd894c3373f7f07bad7161e4b223/readthedocs/redirects/utils.py#L60
-        redirect_filename = filename
-        if lang_slug and version_slug:
-            redirect_filename = '/' + filename
+        # We parse ``filename`` to remove the query from it
+        schema, netloc, path, params, query, fragments = urlparse(filename)
+        filepath = path
```

Comment:
I change this name here to make it more clear. I would expect that `filename=configuration.html`, but this is `filepath=/section/configuration.html`.

I think is a little more clear now, but I'm open for suggestions.

Reply:
we use `filename` for this everywhere, so we should keep it consistent.